src/lambdas/breeds-get.ts

```diff
@@ -0,0 +1,22 @@
+import { getDogBreeds } from '../services/http-req/dogBreedRequest'
+import { flattenObjectValueKeyOrder } from '../services/utilities/flattenObjectValueKeyOrder'
+import { RequestedDogBreeds, ErrorResponse, SuccessResponse } from './types'
+
+export const getBreeds = async (): Promise<SuccessResponse<string[]> | ErrorResponse> => {
+  try {
+    const apiData: RequestedDogBreeds = await getDogBreeds()
+    if (apiData.status === 'error') {
+      throw new Error('Something went wrong')
+    }
```

Comment:
Yes you're right, I totally tried to over engineer getDogBreeds and it made the lambda more complex than it should have been. I removed the try catch from getDogBreeds so it will either return data, or result in an error. This code block is gone now